configure.ac

```diff
@@ -23,10 +23,10 @@
 
 AC_PREREQ([2.68])
 
-AC_INIT([bgpstream], [1.1.0], [bgpstream-info@caida.org])
+AC_INIT([bgpstream], [1.2.0], [bgpstream-info@caida.org])
 
 BGPSTREAM_MAJOR_VERSION=1
-BGPSTREAM_MID_VERSION=1
+BGPSTREAM_MID_VERSION=2
 BGPSTREAM_MINOR_VERSION=0
 
 LIBBGPSTREAM_MAJOR_VERSION=2
```

Comment:
This was a really unfortunate decision of ours to completely separate the libbgpstream version from the bgpstream version.
Unfortunatelyif we increase this number to 3.0.0 (the most sensible thing to do), then we'd have to change the version number of libbgpstream (the one for BGPStream v2) to something higher than 3.

What a nightmare.